mne/viz/_brain/_brain.py

```diff
@@ -409,6 +418,9 @@ def add_data(self, array, fmin=None, fmid=None, fmax=None,
         clim : dict
             Original clim arguments.
         %(src_volume_options_layout)s
+        colorbar_kwargs : dict | None
```

Comment:
Do we have a plan with `colorbar_kwargs`? I don't see it used in this PR.

Reply:
Indeed I don't use it in this example, locally I use it to set the tick and title sizes. Do you think I should update an example?

Reply:
Yes, it would be great to have it used once at least.

Reply:
https://21662-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/source-modeling/plot_visualize_stc.html